packages/gatsby-cli/src/create-cli.js

```diff
@@ -346,6 +346,21 @@ module.exports = argv => {
         }
       ),
     })
+    .command({
+      command: `plugin`,
+      desc: `Welcome to the Gatsby source plugin help center!
+
+      Resources:
+- Walk through the tutorial (https://www.gatsbyjs.org/docs/source-plugin-tutorial/)
+- Read requirements for Gatsby packages (file and naming conventions) (https://www.gatsbyjs.org/docs/how-plugins-work/)
```

Comment:
This file is changing as part of #13261, so we'll probably want to wait until that lands and update this URL.

Reply:
@marcysutton we can also create a short-link, e.g. `https://gatsby.dev/how-plugins-work` and probably _should_ for these types of links!

Reply:
ooh yes! the content referenced on that page has moved though. I think the best place will be: `/docs/creating-plugins/`

And `/docs/creating-a-source-plugin/` or `/docs/pixabay-source-plugin-tutorial/` for the tutorial link.

Reply:
@marcysutton do you have a PR id for the page `/docs/creating-plugins/` ?

Reply:
It was in #13261, which has now been merged! I'll chat with the team about creating short links.